index.js

```diff
@@ -51,6 +46,7 @@ module.exports = {
       config: { baseUrl: `http://localhost:${port}` },
       env: {
         APPLITOOLS_BATCH_ID: 'visual diff',
```

Comment:
I missed this in the last review. No need for an `APPLITOOLS_BATCH_ID` to be passed.

Reply:
removed